lib/SILOptimizer/IPO/CrossModuleOptimization.cpp

```diff
@@ -442,7 +442,9 @@ bool CrossModuleOptimization::shouldSerialize(SILFunction *function) {
   if (function->hasSemanticsAttr("optimize.no.crossmodule"))
     return false;
 
-  if (SerializeEverything)
+  // In embedded Swift we serialize everything.
```

Comment:
We should only serialize all generic functions. There is no need to serialize non-generic functions.
(For performance reasons it's beneficial to serialize small non-generic functions, but it's not required)